website/test/functional/test_views.py

```diff
@@ -0,0 +1,58 @@
+from lxml import html
+from django.urls import reverse
+
+
+def test_login(db, client, data):
+    response = client.post(
+        '/login/', {'username': 'test', 'password': 'test'}
+    )
+    assert response.status_code == 302
```

Comment:
Add follow redirects and check for response code 200

Reply:
`redirect = HttpResponse(response)`
`assert redirect.status_code == 200`